datahub/detector/dtree/url_fetcher.py

```diff
@@ -0,0 +1,21 @@
+class UrlFetcher(object):
+    def __init__(self):
+        self.load_rules()
+
+    def load_rules(self):
+        None
```

Comment:
this should be pass or raise NotImplementedError exception if you have not implemented yet.